src/update.c

```diff
@@ -227,6 +227,12 @@ int do_op(PROGRAMMER * pgm, struct avrpart * p, UPDATE * upd, enum updateflags f
     return -1;
   }
 
+  char alias_mem_desc[65] = "";
+  if(avr_find_memalias(p, mem)) {
+    strcat(alias_mem_desc, "/");
```

Comment:
It would make sense to cache the result of calling `avr_find_memalias()`.
After all, it's a list walk.

Reply:
You mean like this?

``` c
  AVRMEM_ALIAS * alias_mem = avr_find_memalias(p, mem);
  char alias_mem_desc[AVR_DESCLEN + 1] = "";
  if(alias_mem) {
    strcat(alias_mem_desc, "/");
    strcat(alias_mem_desc, alias_mem->desc);
  }
```

Reply:
> You mean like this?

Yep. Might save a few CPU cycles.

Reply:
I like your mindset!